meta-balena-common/recipes-core/initrdscripts/files/finish

```diff
@@ -19,6 +19,9 @@ finish_run() {
         mount --move /proc $ROOTFS_DIR/proc
         mount --move /sys $ROOTFS_DIR/sys
 
+        # Kill watchdog before running systemd. This variable is exported from watchdog module
+        [ -n "$INITRAMFS_WATCHDOG_PID" ] && kill "$INITRAMFS_WATCHDOG_PID" || true
```

Comment:
I'm wondering if there is a way to pet the watchdog before killing it here?
e.g. If we kill the watchdog process when the time is ~ 31 seconds. And systemd hasn't kicked in, we might still get a reboot?

Reply:
I agree. We should pet and kill.

Reply:
I forget to explain what `watchdog -t 10 -T 60` means. `watchdog` command is a very simple infinite loop program that periodically resets the timer. `-T 60` means that it reconfigure the watchdog's deadline to 60 seconds using `WDIOC_SETTIMEOUT` ioctl (60 seconds is Busybox's default), and `-t 10` means that it resets the timer every 10 seconds.
Therefore, systemd has at least 50 seconds in worst case. So I guess we don't need to manually pet the watchdog, if 50 seconds sounds enough.